src/models/AppController.js

```diff
@@ -0,0 +1,86 @@
+import _ from 'lodash';
+import { Logger } from 'zos-lib';
+import { parseJsonIfExists, writeJson } from "zos-lib/lib/utils/FileSystem";
+import { ContractsProvider, AppManagerProvider, AppManagerDeployer } from "zos-lib";
+import StdlibProvider from './stdlib/StdlibProvider';
+import StdlibDeployer from './stdlib/StdlibDeployer';
+import Stdlib from './stdlib/Stdlib';
+import NetworkAppController from './NetworkAppController';
+
+const log = new Logger('AppController');
+
+const DEFAULT_VERSION = '0.1.0';
+
+export default class AppController {
+  constructor(packageFileName) {
+    this.packageFileName = packageFileName || 'package.zos.json';
```

Comment:
what do you think about moving the default to the constructor declaration? I mean
```js
constructor(packageFileName = 'package.zos.json') { ...
```